### PR TITLE
bibledit: initial inclusion

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**
- Initial inclusion.
- Resolves https://github.com/getsolus/packages/issues/2518

**Test Plan**
- Installed and connected to the demo server.

**Checklist**

- [X] Package was built and tested against unstable

### CODE REVIEW

**Review by EbonJaeger — COMMENTED** [review]
One question. Looks fine, otherwise.

**Comment by androidnisse** [comment]
> One question. Looks fine, otherwise.

Question away. :D

File: packages/b/bibledit/package.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    Bibledit is the name of a professional Bible Translation Software Program. It is a collection of Bible translation tools, all conveniently in one place. Online as well as offline access is an important and handy feature. One can save the work offline, and later upload all to the Cloud when there is an active internet connection.
+builddeps  :
+    - pkgconfig(gtest)
+    - pkgconfig(gtk+-3.0)
+    - pkgconfig(icu-i18n)
+    - pkgconfig(libutf8proc)
+    - pkgconfig(libxml-2.0)
+    - pkgconfig(pugixml)
+    - pkgconfig(webkit2gtk-4.0)
+    - pkgconfig(webkit2gtk-4.1)
```
Comment (EbonJaeger) [inline packages/b/bibledit/package.yml:20]: Apparently my message didn't get saved last time. Does this actually need both webkit versions? Only 4.1 appears in `abi_used_libs`.

Comment (androidnisse) [reply packages/b/bibledit/package.yml]: It built fine with only 4.1

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**